_i18n/en.yml

```diff
@@ -124,14 +124,42 @@ sponsorships:
   macstadium: MacStadium sponsors a dedicated Mac Mini that is used for Mac-related development.
 
 team:
-  core: Core
-  developers: Developers
-  developers_para1: The Monero Project has had well over 500 contributors over the life of the project. For a complete list, please see the
-  developers_para2: OpenHub contributors page.
-  developers_para3: Below you'll find some individuals that have gone above and beyond for Monero.
-  community: Community
+  introduction: Workgroups are formed by community members who join forces to achieve a common goal. They have different structures and goals. On this page we list some of the more active workgroups and some tips to get you started, if you wish to contribute to any of them.
+  core: Core Team
+  community: Community Workgroup
   mrl: Research Lab
-  thanks: Special Thanks
+  descr: Description
+  start: Where to start
+  contacts: Chatrooms and Contacts
+  chat: "Chat:"
+  website: "Website:"
+  core_descr: The members of this workgroup are always 7. They are the stewards of the community and they manage and maintain the Monero codebase and infrastructure. The Core Team act as mediators, weighting consensus in the community and taking decisions according to community's will.
+  core_start: The Core Team is a closed workgroup, it's not possible to join it unless invited.
+  community_descr: Workgroup used by the community to organize and coordinate. This is the place where every non-coder who wants to contribute to Monero should start.
+  community_start: The best way to start contributing to this workgroup is to join the chatrooms and introduce yourself, stating your interests and the skills you would like to offer to the community.
+  devworkgroup: Development Workgroup
+  devworkgroup_descr: This workgroup reunites Monero developers working on the wallets and daemon maintained by the Core Team. They are mostly divided in GUI and CLI devs and they coordinate using two different chatrooms.
```

Comment:
"in GUI and CLI devs" is awkward wording. Maybe change the last sentence to "This workgorup consists of two main parts: the CLI and the GUI teams. These teams coordinate using two different chatrooms."